docker/Dockerfile

```diff
@@ -115,6 +112,8 @@ RUN mkdir -p /var/share/TPX08_atlas_compact/ \
 # Patch env when needed here
 RUN pip install --no-cache-dir nbresuse \
   && jupyter serverextension enable --py --system nbresuse \
+  && jupyter serverextension enable --py jupyterlab_code_formatter \
```

Comment:
Please make them all `--system`.
These just write some yaml files, `--system` writes for "all users" somewhere in `/` and without writes in home folder, but home folder is overwritten by volume during deployment, so has to be system

Reply:
ok